tests/test_xopen.py

```diff
@@ -21,7 +23,7 @@
     PipedPythonIsalWriter,
     _MAX_PIPE_SIZE,
     _can_read_concatenated_gz,
-    igzip,
+    igzip
```

Comment:
Please leave in the trailing comma. Trailing commas reduce diff size. We don’t use [Black](https://black.readthedocs.io/en/stable/the_black_code_style.html), yet, but it would add the trailing comma.

Reply:
Done.